packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -10,3 +17,43 @@ export const metadata: ConnectorMetadata = {
   description:
     '邮件推送（DirectMail）是款简单高效的电子邮件群发服务，构建在阿里云基础之上，帮您快速、精准地实现事务邮件、通知邮件和批量邮件的发送。',
 };
+
+interface Template {
+  type: keyof EmailMessageTypes;
+  subject: string;
+  content: string; // With variable {{code}}, support HTML
+}
+
+export interface AliyunDmConfig {
+  accessKeyId: string;
+  accessKeySecret: string;
+  accountName: string;
+  fromAlias?: string;
+  templates: Template[];
+}
+
+export const sendMessage: EmailSendMessageFunction = async (address, type, data) => {
+  const config: AliyunDmConfig = await getConnectorConfig<AliyunDmConfig>(
+    metadata.id,
+    metadata.type
+  );
+  const template = config.templates.find((template) => template.type === type);
```

Comment:
wondering, since we passed in `metadata.type` as an input param to the `getConnectorConfig` util method. Should we keep the template filter by type logic within the method itself?  Instead of returning all the templates?

Reply:
Actually, "templates" is this specific connector's private stuff, contrubitors may provider another `aliyun-dm` connector without this "templates" design.
So is a bit different from `getConnectorConfig`. `getConnectorConfig` is a common util for all connectors.